doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -346,8 +346,13 @@ explicitly in the ``.compopts`` or ``.execopts`` file for the test.
   will then apply for CHPL_COMM != none)
 - ``.no-local.good``: used with ``--no-local`` testing
 - ``.lm-numa.good``: used with CHPL_LOCALE_MODEL=numa
+- ``.na-ofi.good``: used with CHPL_NETWORK_ATOMICS=ofi
```

Comment:
In practice this is always `.na-none.good`. i.e. the default tests with network atomics, and this overwrites to check without network atomics. Comm counts / test output should be the same for all comm layers that support network atomics, so I wouldn't expect to ever use na-ofi or na-ugni or something.